sdk/synapse/Azure.Analytics.Synapse.Shared/tests/SynapseTestExtensions.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+
+namespace Azure.Analytics.Synapse.Tests
+{
+    /// <summary>
+    /// Extension methods to make tests easier to author.
+    /// </summary>
+    public static partial class TestExtensions
+    {
+        /// <summary>
+        /// Convert an IAsyncEnumerable into a List to make test verification
+        /// easier.
+        /// </summary>
+        /// <typeparam name="T">The item type.</typeparam>
+        /// <param name="items">The sequence of items.</param>
+        /// <returns>A list of all the items in the sequence.</returns>
+        public static async Task<IList<T>> ToListAsync<T>(this IAsyncEnumerable<T> items)
```

Comment:
I think there may be something in the test framework that does this....

Reply:
I found a few copies, but none in a common location:

```
PS C:\Users\chhamo\Programming\azure-sdk-for-net\sdk> git grep ToListAsync | Select-String Task

batch/Microsoft.Azure.Batch.FileStaging/tests/FileStagingIntegrationTests.cs:                List<CloudTask> tasksFromService = await
jobOperations.ListTasks(jobId).ToListAsync().ConfigureAwait(false);
batch/Microsoft.Azure.Batch/src/PagedEnumerableExtensions.cs:        public static async Task<List<T>> ToListAsync<T>(this IPagedEnumerable<T> source,
CancellationToken cancellationToken = default(CancellationToken))
batch/Microsoft.Azure.Batch/tests/IntegrationTests/AddTaskCollectionIntegrationTests.cs:                List<CloudTask> tasksFromService = await
jobOperations.ListTasks(jobId).ToListAsync().ConfigureAwait(false);
batch/Microsoft.Azure.Batch/tests/UnitTests/PagedCollectionUnitTests.cs:        public async Task IfSequenceIsNull_ThenToListAsyncThrowsArgumentNullException()
batch/Microsoft.Azure.Batch/tests/UnitTests/PagedCollectionUnitTests.cs:        public async Task ToListAsyncCollectsAllElementsOfSequence()
batch/Microsoft.Azure.Batch/tests/UnitTests/PagedCollectionUnitTests.cs:        public async Task ToListAsyncCorrectlyHandlesEmptyCollections()
batch/Microsoft.Azure.Batch/tests/UnitTests/PagedCollectionUnitTests.cs:        public async Task ToListAsyncSupportsCancellation()
search/Azure.Search.Documents/tests/Utilities/TestExtensions.cs:        public static async Task<List<T>> ToListAsync<T>(this AsyncPageable<T> pageable)
storage/Azure.Storage.Common/tests/Shared/TestExtensions.cs:        public static async Task<IList<T>> ToListAsync<T>(this IAsyncEnumerable<T> items)
synapse/Azure.Analytics.Synapse.Shared/tests/SynapseTestExtensions.cs:        public static async Task<IList<T>> ToListAsync<T>(this IAsyncEnumerable<T> items)
```